### DISCUSSION TITLE
Inquiry About Supervisor Agent Use Case in Autogen Documentation

### DISCUSSION THREAD

**Discussion by mohan-enmovil** [microsoft/autogen#4281] [Q&A]
Hi,
I went through the Autogen documentation but couldn't find a use case for the supervisor agent (A supervisor agent identifies the right agent for a task, assigns it to them, and shares the agent's response with the user).

If this feature is supported, could you please point me to where it's described in the documentation?

Thanks!

**Comment by ekzhu** [comment]
For this I think you can use the Swarm team. It will be available in the next dev release (dev7). 

See the documentation (preview): https://microsoft.github.io/autogen/dev/user-guide/agentchat-user-guide/tutorial/swarm.html#stock-research-example

**Reply by mohan-enmovil** [reply]
Thank you for your response!
I appreciate the information about the Swarm team and the documentation preview. Could you kindly share the expected release date for dev7?

**Reply by ekzhu** [reply]
It has been released now.

**Reply by mohan-enmovil** [reply]
thank you
